app/test_main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+from unittest.mock import MagicMock, patch
+from app.main import can_access_google_page
+
+
+@patch("app.main.has_internet_connection", return_value=True)
+@patch("app.main.valid_google_url", return_value=True)
+def test_can_access_google_page_access_granted(mock_valid_url: MagicMock,
+                                               mock_internet: MagicMock) -> None:
+    url = "https://www.google.com"
+    result = can_access_google_page(url)
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/220956591-fb85ae57-8b1c-4882-a3dc-33711ab33ad7.png)
fix this
also no need to create variable url inside every test, make it constant and use everywhere

Reply:
all tests should pass, fix flake8 errors